notes/meetings/2021-12/december-15.md

```diff
@@ -0,0 +1,45 @@
+# December 15, 2021 - Meeting Notes
+
+**Faciliators:**
+
+| Name | Abbreviation |
+| - | - | - |
+| Dominik Wilkowski | DW |
+| Esther Semo | ES |
+| Elise Chant | EC |
+
+## Resources
+
+- [Survey Summary Results](assets/survey-results.pdf)
+- [Survey Full Results](assets/survey-results-full.csv)
+
+## Prior
+
+* Maintainers have read proposed RFCs
+
+## Survey findings
+
+* ES to show results
+
+## Prioritisation of actions related to survey findings
+
+* TODO 
+
+## Creating a skills register
+
+* TODO 
+
+## Maintainers commitments
+
+* Min time spend
```

Comment:
To be part of any of the aforementioned groups you must spend x amount of time on this project.
We should commit to at least 4 meetings per year such as this one where we go through the RFCs and give the project a push. Implementation can then be assigned to a few people between those meetings.